test/fixtures/poro.rb

```diff
@@ -16,8 +16,8 @@ def method_missing(meth, *args)
     end
   end
 
-  def cache_key_with_digest
-    "#{cache_key}/#{FILE_DIGEST}"
+  def digest
+    FILE_DIGEST
   end
```

Comment:
Only a single test is using this.  The method in unnecessary, the test can just use the constant directly.